docs/usage/executor/x.mdx

```diff
@@ -0,0 +1,215 @@
+---
+id: exec
+title: "Exec"
+sidebar_position: 3
+---
+
+The `exec` executor plugin allows you to install and run CLI applications directly from chat (e.g., Slack, Discord, or Mattermost) without any hassle.
+
+It supports interactivity in the form of buttons and dropdowns. You can easily use all your favorite tools without typing, even from your mobile phone!
+
+![x-demo](./assets/demo-x.gif)
+
+:::note
+To shorten the command, the `x` alias is used in all examples. Make sure you have it configured for your Botkube installation.
+
+```yaml
+aliases:
+  x:
+    command: exec
+    displayName: "Exec alias"
+```
+
+In this context, the `x` stands for `extension` or `execution`.
+:::
+
+# Usage
+
+## Install CLI
+
+To install a given CLI binary directly from a chat window, run:
+
+```
+@Botkube x install {source}
+```
+
+For downloading binaries, the `eget` library is used. It supports multiple URL formats and is able to unpack archives and extract binaries from them. For more details, see the documentation on the [`eget`](https://github.com/zyedidia/eget) GitHub repository.
+
+### Examples
+
+- `@Botkube x install https://get.helm.sh/helm-v3.10.3-linux-amd64.tar.gz --file helm`
+- `@Botkube x install github.com/fluxcd/flux2`
+- `@Botkube x install https://github.com/rikatz/kubepug --asset kubepug_darwin_amd64.tar.gz`
+
+## Execute CLI
+
+To execute a CLI command, run:
+
+```
+@Botkube x run {command}
+```
+
+### Examples
+
+- `@Botkube x run helm list -A`
+- `@Botkube x run flux get sources git`
+
+:::tip
+For convenience, you can configure a Botkube alias to simplify running `helm` commands:
+
+```yaml
+aliases:
+  helm:
+    command: x run flux
+    displayName: "Flux CLI alias"
+```
+
+Later, you can simply type `@Botkube flux get sources git`.
+:::
+
+# Templates
+
+The `exec` plugin supports defining templates for executed commands. As a result, you can specify how to process the CLI output or define your own message response.
+
+```yaml
+# An array of templates that define how to convert the command output into an interactive message.
+templates:
+  - trigger:
+      command:
+        # Specifies the prefix of the command that triggers the template and the parser used to parse the output.
+        # If specified, it has higher priority than the regex field.
+        prefix: "helm list"
+        # Specifies the regex that should match a given command
+        # If specified, it has lower priority than the prefix field.
+        regex: "regex: '^helm list(?:\s+(-A|-a))*\s?$'"
+    # Specifies the message template type.
+    type: ""
+    # Message template depending on the selected type.
+    message: { }
+```
+
+Supported template types:
+
+- `parser:table:space`
+- `wrapper`
+- `tutorial`
+
+Navigate to the [`exec`](https://github.com/kubeshop/botkube/tree/main/cmd/executor/x/templates) plugin directory to see official examples.
+
+import Tabs from "@theme/Tabs";
```

Comment:
Now the description of the templates is perfect 👌 Please consider using usual sections (with h2) instead of tabs, as in that way we can link to them directly (and the headers are great for SEO). Thanks!

Reply:
changed back to headers 👍